exercism_test_helper/.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.10.4-otp-23
```

Comment:
Erlang version should also be specified in this file. I was getting an Erlang crash when trying to build this project, and then I noticed my default Erlang is 22.
```suggestion
elixir 1.10.4-otp-23
erlang 23.1
```

On top of that, it's also necessary to have a `.tool-versions` file in the root directory of this project, with just the Erlang version (maybe it's enough to specify the Erlang version just in the root direcctory and it will also be used inside of the `exercism_test_helper` directory? not sure how asdf behaves when there are two `.tool-versions`)

If I build the script with Erlang 23 from the directory `exercism_test_helper` and move it to root, it can't run from the root directory when there's a different Erlang version there. I was getting this error:
```
$ ./bin/run.sh two-fer /Users/angelika/Exercism/elixir/two-fer .
parsing /Users/angelika/Exercism/elixir/two-fer/test/two_fer_test.exs for metadata
escript: exception error: undefined function exercism_test_helper_escript:main/1
  in function  escript:run/2 (escript.erl, line 758)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
```